### PR TITLE
Run on dev and 2.6.0

### PR DESCRIPTION
**Pull request by grouma**
Remove Angular from the critical path and skip tests broken with the dev SDK.

### CODE REVIEW

**Comment by jakemac53** [comment]
This should really be the last supported version + dev

File: .travis.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ services:
 jobs:
   include:
     - stage: analyzer_and_format
-      name: "SDK: 2.6.0; PKGS: dwds, webdev; TASKS: `dartanalyzer --fatal-warnings .`"
-      dart: "2.6.0"
+      name: "SDK: dev; PKGS: dwds, webdev; TASKS: `dartanalyzer --fatal-warnings .`"
+      dart: dev
       os: linux
```
Comment (annagrin) [inline .travis.yml]: I am not very familiar with how analyzer job is run - will running dev analyzer with stable SDK cause any problems? If yes, we might need two different analyzer jobs, depending on what SDK is being used for the test

Comment (grouma) [reply .travis.yml]: Each of these are disjoint jobs so they get the right SDK.

Comment (jakemac53) [reply .travis.yml]: The analyzer is always being used from the SDK itself - so its always the same version.

We do expect to be statically analyzable on all versions of the SDK we claim to support.

Comment (annagrin) [reply .travis.yml]: Thanks for clarification!

**Pull request merged**